extensions/git/package.json

```
@@ -411,6 +411,26 @@
         "category": "Git"
       }
     ],
+    "keybindings": [
+      {
+        "command": "git.stageSelectedRanges",
+        "key": "ctrl+k ctrl+alt+s",
+        "mac": "cmd+k cmd+alt+s",
+        "when": "isInDiffEditor"
+      },
+      {
+        "command": "git.unstageSelectedRanges",
+        "key": "ctrl+k ctrl+u",
+        "mac": "cmd+k cmd+u",
+        "when": "isInDiffEditor"
+      },
+      {
+        "command": "git.revertSelectedRanges",
+        "key": "ctrl+k ctrl+r",
+        "mac": "cmd+k cmd+r",
```

```suggestion
        "mac": "cmd+k cmd+alt+r",
```
macOS VS Code has <kbd>Cmd</kbd><kbd>K</kbd> <kbd>Cmd</kbd><kbd>R</kbd> as a builtin keyboard shortcut that opens this link: https://code.visualstudio.com/shortcuts/keyboard-shortcuts-macos.pdf
I don't know about Linux and windows, but best not to conflict.

---

Perhaps if we do use <kbd>alt</kbd>, probably best to change revert for more uniformity?